sdks/xmtp-cli/README.md

```diff
@@ -0,0 +1,339 @@
+# XMTP CLI
+
+A barebones CLI built with [Commander.js](https://github.com/tj/commander.js) for managing XMTP protocol operations.
+
+## Quick Start
+
+```bash
+# Run commands via yarn or the xmtp binary
```

Comment:
in the main `README`, the instructions should only be for the CLI, not working in the repo. if you want to add a development section, you could outline internal repo commands there.